tests/baselines/reference/user/TypeScript-React-Native-Starter.log

```diff
@@ -1,7 +1,50 @@
-Exit Code: 1
+Exit Code: 0
 Standard output:
-node_modules/@types/react-native/index.d.ts(3895,26): error TS2693: 'setImmediate' only refers to a type, but is being used as a value here.
-node_modules/@types/react-native/index.d.ts(3896,28): error TS2693: 'clearImmediate' only refers to a type, but is being used as a value here.
+Version 3.2.0-dev
```

Comment:
The two typescript-starter diffs are probably caused by the sub module update. Not sure why this one acts like the tsconfig is gone now.